angular/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM node:6.10
+
+MAINTAINER datapunt.ois@amsterdam.nl
+
+EXPOSE 80
+
+
+RUN apt-get update \
+ && apt-get install -y git nginx build-essential openjdk-7-jre \
+ && apt-get clean \
+ && mkdir /app
+
+ENV JAVA_HOME /usr/lib/jvm/java-7-openjdk-amd64
+
+WORKDIR /app
+COPY *.json /app/
+
+RUN rm -rf node_modules\
+ && yarn install
+
+COPY . /app/
```

Comment:
Dit overschrijft de node/modules die net door yarn install is gemaakt. Cross platform gaat dat fout. (ook in Atlas). Deze gaat bijvoorbeeld wel goed: https://github.com/DatapuntAmsterdam/mijn-amsterdam/blob/master/Dockerfile